nautobot/dcim/tests/test_views.py

```diff
@@ -585,6 +585,47 @@ def setUpTestData(cls):
             "Manufacturer 7,Seventh manufacturer",
         )
 
+    @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
+    def test_manufacturer_list_view_count_related_subquery(self):
```

Comment:
Testing the view is adding a lot of unnecessary overhead. Can you test the failing queryset instead?

```py
Manufacturer.objects.annotate(inventory_item_count=count_related(InventoryItem, "manufacturer"))
```

Reply:
Might be better suited as a generic test in fact. This appears to only be affected by tree models

Reply:
I'm not sure how you would add this as a generic test without specifying the model (`Manufacturer`) and the field (`InventoryItem`) in each test. How to detect which FK field is a `TreeNode` model and etc.

Reply:
Sorry, I didn't mean a generic test that applies to multiple models/querysets/views. You could just move this test to a `test_count_related` method in `nautobot/core/tests` and instead of calling the view just call the `Manufacturer.objects.annotate(inventory_item_count=count_related(InventoryItem, "manufacturer"))` queryset?